predictor-server/src/main/resources/datacrawler.properties

```diff
@@ -0,0 +1,7 @@
+alphavantage.url=https://www.alphavantage.co/query
+alphavantage.function=TIME_SERIES_INTRADAY
+alphavantage.symbol=MSFT
```

Comment:
Currently Data Crawler supports only one equity, e.g. MSFT (Microsoft). Could be extended to multiple equities, e.g. "MSFT,AAPL".